crypto/CMakeLists.txt

```diff
@@ -452,13 +452,34 @@ add_library(
   x509v3/v3_skey.c
   x509v3/v3_utl.c
 
-  $<TARGET_OBJECTS:fipsmodule>
-
   ${CRYPTO_ARCH_SOURCES}
-  ${CRYPTO_FIPS_OBJECTS}
 )
 
-target_compile_definitions(crypto PRIVATE BORINGSSL_IMPLEMENTATION)
+target_compile_definitions(crypto_objects PRIVATE BORINGSSL_IMPLEMENTATION)
```

Comment:
minor nit: new line here to make it easier to see the `build_libcrypto` function?